lib/launcher.js

```diff
@@ -33,6 +37,11 @@ define(function (require, exports, module) {
         });
     };
 
+    function getCurrentInstance() {
+        return _launcherInstance;
+    }
+
     // Define public API
     exports.Launcher = Launcher;
+    exports.getCurrentInstance = getCurrentInstance;
```

Comment:
Agreed, you want `Launcher.getCurrentInstance();`, so it would be:

```js
Launcher.getCurrentInstance = function() {
  return _launcherInstance;
};
```
And then you can remove the exports line on 46.